optuna_dashboard/ts/components/AppDrawer.tsx

```diff
@@ -300,6 +303,19 @@ export const AppDrawer: FC<{
                 <ListItemText primary="Note" sx={styleListItemText} />
               </ListItemButton>
             </ListItem>
+            <ListItem key="Settings" disablePadding sx={styleListItem}>
+              <ListItemButton
+                component={Link}
+                to={`${URL_PREFIX}/studies/${studyId}/settings`}
+                sx={styleListItemButton}
+                selected={page === "settings"}
+              >
+                <ListItemIcon sx={styleListItemIcon}>
+                  <SettingsIcon />
```

Comment:
Is it better with [ColorLens](https://mui.com/material-ui/material-icons/)?
Optionally, I would like to discuss if "settings" is an appropriate name for this feature because the name is too general.
If we add any further configurations here in the future, I am definitely for it:)